subsys/nrf_rpc/Kconfig

```diff
@@ -84,6 +84,33 @@ config NRF_RPC_UART_RX_THREAD_STACK_SIZE
 	  thread is responsible for consuming data received over the UART, and
 	  passing decoded nRF RPC packets to the nRF RPC core.
 
+menu "Reliability"
+
+menuconfig NRF_RPC_UART_RELIABLE
+	bool "Enable acknowledgements"
+	help
+	  Enables acknowledgment functionality for each frame sent over UART.
+
+if NRF_RPC_UART_RELIABLE
+
+config NRF_RPC_UART_ACK_WAITING_TIME
+	int "Time window to receive acknowledgment"
+	default 10
```

Comment:
Seems a bit low to me. On nRF52, it may take up to 85ms to erase a single flash page during which the device is not able to process any frames and send the ACK. The waiting time * the number of attempts should be greater than that to survive such stalls. I would suggest the waiting time of >50ms and the number of attempts 3 as a starting point.